great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -500,28 +591,18 @@ def _build_numeric_columns_rule() -> Rule:
                 "lower_bound": None,
                 "upper_bound": None,
             },
-            "round_decimals": 1,
+            "round_decimals": 4,
```

Comment:
This is the only parameter that changes for the high precision rule.

Reply:
If these two methods are largely the same, could we possibly refactor to wrap that logic in a helper function (and just have high/low precision change this number)?.

Reply:
After discussing this with @alexsherstinsky the reason why we can't refactor this to use a single rule with different parameters passed into `NumericMetricRangeMultiBatchParameterBuilder` is because the user needs to have the ability to change the `DataAssistant` behavior at runtime via the `variables` mechanism.